src/sonarcloud.ts

```diff
@@ -25,14 +25,26 @@ export const checkForErrors = (response: any) => {
   }
 };
 
-export const handleErrors = (errors: Error[]) => {
-  for (const error of errors) {
-    if (error.message.includes("No organization for key")) {
+type SonarcloudError = {
+  message?: string;
+  msg?: string;
+};
+
+export const handleErrors = (errors: SonarcloudError[]) => {
+  for (let error of errors) {
+    const errorKeys = Object.keys(error);
+    if (!errorKeys.includes("message") && !errorKeys.includes("msg")) {
+      throw new Error(`Error message not found: ${error}`);
+    }
+    if (errorKeys.includes("msg")) {
+      error.message = error.msg;
+    }
```

Comment:
It's usually a better pattern to pull the data out of the object to operate on rather than modify the object passed in, so something like this:

```suggestion
    const errorMessage = error.message || error.msg;
```

...would mean you can do everything you need below with `errorMessage` rather than `error.message`, and most importantly we know that what we log on lines 43 and 50 are exactly what we got back from the remote API, without us having duplicated any of the contents.